src/Microsoft.Health.Operations/Microsoft.Health.Operations.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We could also call this `Microsoft.Health.Operations.Abstractions`

Reply:
I tried to rename, but I didn't like it when needing the abstractions namespace for `OperationId`